coderd/database/migrations/testdata/fixtures/000189_duplicate_user_links.up.sql

```diff
@@ -0,0 +1,19 @@
+-- This is a deleted user that shares the same username and linked_id as the existing user below.
```

Comment:
@coadler Is this ok to name this fixture different then the last migration?

Should I make this fixture to an earlier migration number?

Reply:
I wonder if you could also just add it in the base fixture 🤔

Reply:
I was wondering that too. Was not sure what the protocol is on the immutability of test fixtures.

Reply:
I would probably add it as the same migration id that `user_links` was added, but I'm not super familiar with these. Maybe @mafredri can weight in

Reply:
Yeah, since the fixtures should never make their way into a customer deployment I think this should be fine.

Reply:
I moved it to `000048_userdelete.up.sql`. That is when deleting users was supported. Which is what this really is, adding a deleted user.

Reply:
Yeah definitely, always feel free to go wild with the fixtures, and use them to place in wrong-use traps for the code. 😄